### ISSUE TITLE
Mapmap won"t start

### ISSUE THREAD

**Issue by tzukowski3101** [mapmapteam/mapmap#521]
Any of available version of Mapmap won"t start (including beta version). After instalation, any attempt of launching this program wasn't succesful. My operation system is Windows 10 Home.
Any ideas why ?

**Comment by aalex** [comment]
Did you install GStreamer?

**Comment by tzukowski3101** [comment]
Yes i did, nothing changed.  Should I do anything else with GStreamer?

**Comment by tzukowski3101** [comment]
I discover the only way to launch mapmap is by using virtualbox with Windows version: 20H2. On my computer with the same version still the same problem.
